README.md

```diff
@@ -208,6 +208,8 @@ module.exports = {
   directusToken: 'my-directus-token',
   directusEmail: 'admin@example.com', // ignored if directusToken is provided
   directusPassword: 'my-directus-password', // ignored if directusToken is provided
+  directusClientConfig: {} // see https://docs.directus.io/guides/sdk/getting-started.html#polyfilling
+  directusRestConfig: {} // see https://docs.directus.io/packages/@directus/sdk/rest/interfaces/RestConfig.html
```

Comment:
You could group those configurations into a single object:

```
directusConfig: {
  clientOptions: {},
  restConfig: {}
};
```

This approach will allow you to extend these configurations later without the risk of expanding the root configuration object excessively.